src/fluree/server/components/http.clj

```diff
@@ -1,30 +1,30 @@
 (ns fluree.server.components.http
   (:require
```

Comment:
Stuart Sierra notwithstanding, I prefer the first ns require to be on the same line as the `:require` keyword. I think the extra indentation that gives helps break up the require blocks vs imports, uses, and refer-clojures.

Reply:
I tend to agree, but I think this is another thing that cljfmt cannot (yet?) enforce.